pages/common.py

```diff
@@ -130,10 +130,33 @@ def body(content):
 <script src="assets/js/custom.js"></script>
 <!-- SLICK CAROUSEL -->
 <script type="text/javascript" src="assets/slick/slick.min.js"></script>
-
+<!-- BOOTBOX -->
+<script type="text/javascript" src="assets/js/bootbox.min.js"></script>
+
+<!-- Chat link modal alert handler, this is the only place it can go at the moment  -->
```

Comment:
Explaining why would be good. Something like:

    <!-- Chat link modal alert handler. This must go after jQuery is loaded, which restricts it to being in common. -->